pkg/apis/core/v1/helper/helpers.go

```diff
@@ -135,16 +135,6 @@ func ingressEqual(lhs, rhs *v1.LoadBalancerIngress) bool {
 	return true
 }
 
-// TODO: make method on LoadBalancerStatus?
```

Comment:
i agree with this comment,
IMO deep copy semantics should be on the side of the API object, instead of having a public method that performs that "somewhere".

Reply:
doesn't LoadBalancerStatus#DeepCopy() already exist?

Reply:
it seems to be generated correctly:
https://github.com/kubernetes/api/blob/master/core/v1/zz_generated.deepcopy.go#L2124

Reply:
I think we can use `deepCopyInto()` and leave `LoadBalancerStatusEqual()` as is